<launch>
    <!-- SYSTEM MODEL -->
    <!-- vehicle parameters -->
    <param name="m" type="double" value="1.98" />
    <param name="I_z" type="double" value="0.24" />
    <param name="L_a" type="double" value="0.125" />
    <param name="L_b" type="double" value="0.125" />
    <!-- control parameters -->
    <param name="input_gain" type="double" value="0.3" />

    <!-- ARDUINO -->
    <!--  * encoders and ultrasound sensors -->
    <!--node pkg="rosserial_python" type="serial_node.py" name="arduino_node" >
        <param name="port" value="/dev/ttyUSB0" /-->
    <!--/node-->

    <!-- LOW LEVEL CONTROLLER -->
    <node pkg="smartcar" name="ControllerLowLevel" type="ControllerLowLevel.py"  output="screen"/>

	<!-- TRANSLATOR -->
    <node pkg="smartcar" name="Translator"  type="Translator.py" output="screen"/>

    <!-- SUBTASKER -->
    <node pkg="smartcar" name="Subtasker" type="Subtasker.py" output="screen"/>

    <!-- TASKIO -->
    <node pkg="smartcar" type="TaskIO.py" name="TaskIO"  output="screen"/>

    

    
</launch>
